client/grafana_http_api_client.go

```diff
@@ -394,15 +394,6 @@ func (c *GrafanaHTTPAPI) WithOrgID(orgID int64) *GrafanaHTTPAPI {
 	return c
 }
 
-// WithRetries sets retry parameters and returns the client
```

Comment:
Can you add this to the client template? My bad. I'm currently adding a CI check to make sure this doesn't happen again

Reply:
Done.